settings.gradle

```diff
@@ -212,6 +212,12 @@ project(':engine-query-constants').projectDir = file('engine/query-constants')
 include(':engine-sql')
 project(':engine-sql').projectDir = file('engine/sql')
 
+include ':engine-processor'
+project(':engine-processor').projectDir = file('engine/processor')
+
+include ':engine-processor-functions'
+project(':engine-processor-functions').projectDir = file('engine/processor-functions')
```

Comment:
Why do we mix `/` and `-`? Should it be in `engine/process/functions`?
Plural or singular function?

Reply:
It's not without precedent; `extensions/source-support`, `engine/test-utils`, `engine/query-constants`.

In my mind, `processor-functions` isn't really hierarchical, it's `engine-processor` + `util-function` at the same level:

```gradle
    api project(':engine-processor')
    api project(':util-function')
```